### DISCUSSION TITLE
Group by date and select group

### DISCUSSION THREAD

**Discussion by NicolasRannou** [petyosi/react-virtuoso#285] [Q&A] [answered]
Hello,

I'm trying to build a google photo like app.

Reading at the documentation and examples, it was not clear to me if it was possible to choose how data is grouped.

I've got a lot of data and I want to group it by date.
On top of that, I'd like to select all the data available in a group.

Is such things possible with virtuoso?

Awesome work and looking forward to hear from you,
Cheers

**Comment by petyosi** [comment, marked as answer]
The short answer is yes - this scenario is supported by the `GroupedVirtuoso` component. The UI component is not concerned about the shape of the data rendered. You can build your groups up to your requirements and manage any additional selection state.
